invokeai/backend/model_manager/util/model_util.py

```diff
@@ -159,3 +159,8 @@ def convert_bundle_to_flux_transformer_checkpoint(
         del transformer_state_dict[k]
 
     return original_state_dict
+
+
+@torch.jit.script
+def convert_sd_entry_to_bfloat16(key: str, tensor: torch.Tensor) -> tuple[str, torch.Tensor]:
+    return key, tensor.to(torch.bfloat16, copy=False)
```

Comment:
In case it's not clear, `copy=False` is the default behaviour, and a copy is still necessary if the dtype is being changed. Just wanted to call that out since it seemed unusual to explicitly set `copy=False`.